src/main/java/frc/robot/OI.java

```diff
@@ -34,8 +34,8 @@ public static final class Driver {
     // Bumpers & Triggers
     public static final Control intakeTrigger =
         new Control(XboxController.Axis.kLeftTrigger, "Run intake", controller, 0.5);
-    public static final Control RT =
-        new Control(XboxController.Axis.kRightTrigger, null, controller, 0.5);
+    public static final Control point_forward =
```

Comment:
Ideally all of these constants would follow the `UPPER_CASE_UNDERSCORES` convention, but since the others in this file use `camelCase`, this one should, too.